react/features/toolbox/components/web/Toolbox.js

```diff
@@ -376,6 +388,11 @@ class Toolbox extends Component<Props> {
                         visible = { this._shouldShowButton('camera') } />
                 </div>
                 <div className = 'button-group-right'>
+                    <LocalRecordingButton
+                        isDialogShown = { this.props._localRecState.showDialog }
+                        onClick = {
+                            this._onToolbarToggleLocalRecordingInfoDialog
+                        } />
```

Comment:
Check out how the other buttons are gated behind some configs. Your feature must do the same, not just for consistency but so we can keep it disabled while testing and prevent it from getting displayed on certain environments.

Reply:
Also, what was the browser compatibility on local recording? It's useable across all browsers?